deltachat-ios/Controller/AccountSetup/WelcomeViewController.swift

```diff
@@ -218,11 +218,13 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
     }
 
     @objc private func moreButtonPressed() {
-        let alert = UIAlertController(title: "Encrypted Account (experimental)",
-                                      message: "Do you want to encrypt your account database? This cannot be undone.",
+        let alert = UIAlertController(title: "Encrypt Database (highly experimental, use at your own risk)",
+                                      message: "Encrypting your database will slow down your accout and notifications. "
```

Comment:
```suggestion
                                      message: "Encrypting your database will slow down your account and notifications. "
```

Reply:
thanks, i slightly reworded and say "the app and notifications"